### PR TITLE
bots: Finish py2 -> py3 tasks for now.

### PR DESCRIPTION
**Pull request by showell**
@roberthoenig Please feel free to merge this if it builds.

### CODE REVIEW

**Comment by roberthoenig** [comment]
@showell I"m wondering if the second commit should close #187, since we haven't moved all code using `six` yet.

File: tools/provision [hunk 1]
```diff
@@ -40,7 +40,14 @@ the Python version this command is executed with."""
...
     venv_name = 'zulip-api-py{}-venv'.format(py_version[0])
 
+    if py_version <= (3, 1) and (not options.force):
+        print(red + "Provision failed: Cannot create venv with outdated Python version ({}).\n"
+              "Run this script with --force if you want to proceed anyway."
+              .format(py_version_output.strip()) + end_format)
+        sys.exit(1)
+
     venv_dir = os.path.join(base_dir, venv_name)
+    print(venv_dir)
```
Comment (roberthoenig) [inline tools/provision]: Should this be part of the commit?

**Comment by roberthoenig** [comment]
Nevermind, just read your comment on the issue.

**Comment by showell** [comment]
blockquote, div.yahoo_quoted { margin-left: 0 !important; border-left:1px #715FFA solid !important; padding-left:1ex !important; background-color:white !important; } Oops that was a stray print.

On Sunday, December 31, 2017, 10:49 AM, Robert Hönig <notifications@github.com> wrote:


@roberthoenig commented on this pull request.

In tools/provision:
>      venv_dir = os.path.join(base_dir, venv_name)
+    print(venv_dir)

Should this be part of the commit?

**Comment by showell** [comment]
@roberthoenig I updated the PR to remove the stray print and fix the lint error.

**Comment by roberthoenig** [comment]
Merging, thanks @showell! (This feels weird, haha)

**Comment by showell** [comment]
You'll get used to it. :smile: Thanks for merging!

**Pull request merged**